zulipterminal/core.py

```diff
@@ -34,8 +34,7 @@ def __init__(self, config_file: str, theme: ThemeSpec,
         self.autohide = autohide
         self.notify_enabled = notify
 
-        self._editor_mode = False
-        self._editor = None  # type: Any
+        self._editor = None  # type: Optional[Any]
```

Comment:
If I understand this right, we are doing away with `self._editor_mode` altogether. Maybe, add a small note in commit description about this?